### PR TITLE
add Home page

### PR DESCRIPTION
**Pull request by hsnice16**
added design for home page

Live: [https://poshui-documentation.netlify.app/](https://poshui-documentation.netlify.app/)

### CODE REVIEW

**Comment by anik31** [comment]
LGTM

**Review by Maianki — APPROVED** [review]
Looks good to me.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            </li>
+            <li class="nav-link margin-lr-1">
+              <a href="#">Documentation</a>
+            </li>
+            <li class="nav-link margin-lr-1">
+              <a
+                href="https://github.com/hsnice16/PoshUI-Documentation"
+                target="_blank"
+                >GitHub</a
+              >
```
Comment (samyukthasudhakar) [inline index.html:60]: Closing tag '>' indent has been separated to a new line.

**Review by samyukthasudhakar — APPROVED** [review]
The logo design looks fab. Clean and amazing UI. LGTM !

**Comment by Developers-codz** [comment]
Looks awesome

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        <a href="#">
+          <button class="btn btn-primary-solid btn-squared btn-home">
+            Get Started
+          </button>
+        </a>
+
+        <a
+          href="https://github.com/hsnice16/PoshUI-Documentation"
+          target="_blank"
+        >
```
Comment (vrx29) [inline index.html:81]: anchor tag is not properly formatted. Everything else looks good

File: index.html (hunk 1 above)
Comment (hsnice16) [reply index.html:60]: May be, It is because of prettier. But thanks for pointing it, Will look into it.

Comment (hsnice16) [reply index.html:60]: I checked this, prettier is doing. So, I think, I should let it be like this.

File: index.html (hunk 2 above)
Comment (hsnice16) [reply index.html:81]: Hey, I checked this, prettier is doing. So, I think, I can leave it like this.

**Pull request merged**